README.md

```diff
@@ -1 +1,79 @@
-# Online-book-store
+# "Online Bookstore"
+
+---
+
+### Project description:
+
+'Online Bookstore' is a web application that simplifies book selection and purchase. Users can easily add, store, select, and buy books from various categories without visiting a physical store. The application utilizes Java-based server technologies like the Spring Framework, JWT, and Docker.
+
+Online Book Store API offers comprehensive book and category management, order placement, and shopping cart features. It ensures a seamless experience for book enthusiasts and administrators, covering everything from book shopping to store management.
+
+---
+
+### Technologies
+- Java 17
+- Spring Boot, Spring Security, Spring data JPA
+- REST, Mapstruct
+- MySQL, Liquibase
+- Maven, Docker
+- Lombok, Swagger
+- Junit, Mockito, testcontainers
+---
+
+### Current project functionality
+
+The application has USER or ADMIN roles. The USER role is assigned automatically to each new user. You can also view the list of all application endpoints.
+
```

Comment:
a bit of information on project functionality. 

add what have done
what kind of actions can make the USER or ADMIN